modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -387,3 +397,7 @@ def empty(cls):
             New `PandasDataframePartition` object.
         """
         return cls.put(pandas.DataFrame(), 0, 0)
+
+    @property
+    def _is_debug(self):  # noqa: GL08
+        return get_logger().isEnabledFor(logging.DEBUG)
```

Comment:
could you also please measure that using this construction is not heavier than building the identity itself

Reply:
now yes, `get_logger()` call was heavy enough.

```bash
   ncalls  tottime  percall  cumtime  percall filename:lineno(function)
   118091    0.067    0.000    0.102    0.000 ...pandas/partitioning/partition.py:401(_is_debug)
```

Reply:
so now it's better? what are the numbers showing in your comment, did you measure the timings before or after you made optimizations regarding getting a single logger object for a single method?

Reply:
Yes, it's better. These are the results after optimization, it became 0.5 seconds faster. The line with the results above is taken from the profile collected for one of the workflows (not a synthetic example).